runtime/browser/web_application.h

```diff
@@ -105,11 +105,18 @@ class WebApplication : public WebView::EventListener {
   void SetupWebView(WebView* view);
   void RemoveWebViewFromStack(WebView* view);
 
+  void SetWindowOrientation();
+  void SetSplashScreen();
+
+  void CreateWindow(common::AppControl* appcontrol);
+  void RemoveWindow();
+  NativeWindow::WindowCategory GetWindowCategory(common::AppControl* ac);
```

Comment:
It must not be static, because it uses private data of WebApplication class instance.